### PR TITLE
Add support for UBI 9 Docker images for main branch (nightly)

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

File: nightly-main/rhel-ubi/9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+RUN swift --version
+
+RUN echo '[ ! -z "$TERM" -a -r /etc/motd ] && cat /etc/motd' \
+    >> /etc/bashrc; \
+    echo -e " ################################################################\n" \
+    "#\t\t\t\t\t\t\t\t#\n" \
+    "# Swift Nightly Docker Image\t\t\t\t\t#\n" \
+    "# Tag: $(cat .swift_tag)\t\t\t#\n" \
+    "#\t\t\t\t\t\t\t\t#\n"  \
+    "################################################################\n" > /etc/motd
```
Comment (MaxDesiatov) [inline nightly-main/rhel-ubi/9/Dockerfile:66]: formatting nit: missing a newline at the end of the file

File: nightly-main/rhel-ubi/9/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  python3-devel     \
+  rsync             \
+  sqlite-devel      \
+  unzip             \
+  zip
+
+# Everything up to here should cache nicely between Swift versions, assuming dev dependencies change little
+
+# gpg --keyid-format LONG -k FAF6989E1BC16FEA
+# pub   rsa4096/FAF6989E1BC16FEA 2019-11-07 [SC] [expires: 2021-11-06]
```
Comment (MaxDesiatov) [inline nightly-main/rhel-ubi/9/Dockerfile]: This looks already expired? `[expires: 2021-11-06]`

Comment (shahmishal) [reply nightly-main/rhel-ubi/9/Dockerfile]: Fixed the copy/paste issue

**Comment by shahmishal** [comment]
@swift-ci test

**Pull request merged**